### PR TITLE
cast positional arguments to int

### LINKED ISSUE #14732: Basic Drawing Tutorial - python: error
##### System information (version)

- OpenCV => 4.1.0
- Operating System / Platform => Mac OS Mojave 10.14
- Compiler => n/a python 3.7

##### Detailed description

https://docs.opencv.org/4.1.0/d3/d96/tutorial_basic_geometric_drawing.html
python version of tutorial throws the following error:

'''TypeError: ellipse() takes at most 5 arguments (9 given)'''

This error appears to be a bit misleading.  Instead the problem seems to be the way python 3 treats integer division (converts to float).  Casting positional arguments to int should solve this.  

##### Steps to reproduce

```.py
    cv.ellipse(img,
                (W / 2, W / 2),
                (W / 4, W / 16),
                angle,
                0,
                360,
                (255, 0, 0),
                thickness,
                line_type)

### PR DESCRIPTION
**Pull request by SchultzC**
resolves #14732
### This pullrequest changes
Fixes drawing errors by casting positional arguments to int

### CODE REVIEW

File: samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py [hunk 1]
```diff
@@ -6,10 +6,10 @@
 def my_ellipse(img, angle):
     thickness = 2
     line_type = 8
-
+    
     cv.ellipse(img,
-                (W / 2, W / 2),
-                (W / 4, W / 16),
+                (int(W / 2), int(W / 2)),
+                (int(W / 4), int(W / 16)),
```
Comment (dkurt) [inline samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py]: `W // 2` is better (integer division).

Comment (SchultzC) [reply samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py]: Sounds good.  Changed based on feedback.

File: samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
 def my_ellipse(img, angle):
     thickness = 2
     line_type = 8
-
+
```
Comment (dkurt) [inline samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py]: Please try to remove these trailing whitespace.

Comment (SchultzC) [reply samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py]: White spaces removed.

**Pull request merged**